src/component/admin/apiToken/ApiTokenForm/SelectProjectInput/SelectAllButton/SelectAllButton.tsx

```diff
@@ -15,7 +15,11 @@ export const SelectAllButton: FC<SelectAllButtonProps> = ({
 
     return (
         <Box sx={{ ml: 3.5, my: 0.5 }}>
-            <Link onClick={onClick} className={styles.selectOptionsLink}>
+            <Link
+                onClick={onClick}
+                className={styles.selectOptionsLink}
+                role="button"
```

Comment:
Looks like this results in an `<a role="button">`. We should instead just use a `<button>`.